### PR TITLE
Added new actor(s) for bollywood to survive.!

### PR DESCRIPTION
**Pull request by MGrow**
Please review this, @casey-collab , and help me contribute!!! :D

### CODE REVIEW

File: hindi/actor4/aamir.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+आप अमीर खान की गोद में खेल रहे हैं|  मज़ा आ रहा है ?
```
Comment (casey-collab) [inline hindi/actor4/aamir.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: hindi/actor4/deepika.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+अब आप दीपिका की गोद में हैं |
```
Comment (casey-collab) [inline hindi/actor4/deepika.md]: This file has also been added to a directory that already contained a page.

File: hindi/bollywood.md [hunk 3]
```diff
@@ -6,4 +6,6 @@ SRK ,[go here](actor1/srk.md)
 
 Salman,[go here](actor2/salman.md)
 
-Amithab, [go here](actor3/amithab.md)
\ No newline at end of file
+Amithab, [go here](actor3/amithab.md)
+
+आमिर खान तो भूल ही गए ये लोग! [यहाँ दबाये] (actor4/aamir.mdamir.md)
```
Comment (casey-collab) [inline hindi/bollywood.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by MGrow** [comment]
@casey-collab, check now?

**Comment by casey-collab** [comment]
Looks great! :ribbon:

**Pull request merged**